plugins/gcloud/gcloud.plugin.sh

```diff
@@ -0,0 +1,49 @@
+#! bash oh-my-bash.module
+# gcloud.plugin.sh
+# Author: Antonino Cangialosi (github.com/ninoCan)
+# Fork of oh-my-bash gcloud plugin
+
+if [[ -z "${CLOUDSDK_HOME}" ]]; then
+    search_locations=(
+        "$HOME/google-cloud-sdk"
+        "/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk"
+        "/opt/homebrew/Caskroom/google-cloud-sdk/latest/google-cloud-sdk"
+        "/usr/share/google-cloud-sdk"
+        "/snap/google-cloud-sdk/current"
+        "/usr/lib/google-cloud-sdk"
+        "/usr/lib64/google-cloud-sdk"
+        "/opt/google-cloud-sdk"
+        "/opt/local/libexec/google-cloud-sdk"
+    )
+
+    for gcloud_sdk_location in $search_locations; do
+        if [[ -d "${gcloud_sdk_location}" ]]; then
+            CLOUDSDK_HOME="${gcloud_sdk_location}"
+            break
+        fi
+    done
+    unset search_locations gcloud_sdk_location
+fi
+
+if [[ -n $CLOUDSDK_HOME ]]; then
+    which gcloud &> /dev/null
+    # Only source this if gcloud isn't already on the path
+    if [[ $? -ne 0 ]]; then
+        if [[ -f "${CLOUDSDK_HOME}/path.bash.inc" ]]; then
+            source "${CLOUDSDK_HOME}/path.bash.inc"
+        fi
+
+        # Look for completion file in different paths
+        completion_filepath_candidates=(
+            "${CLOUDSDK_HOME}/completion.bash.inc"             # default location
+            "/usr/share/google-cloud-sdk/completion.bash.inc"  # apt-based location
+        )
+        for comp_file in "$completion_filepath_candidates"; do
+            if [[ -f "$comp_file" ]]; then
+                source "$comp_file"
+                break
+            fi
+        done
+        unset comp_file
+    fi
+fi
```

Comment:
Please add a newline at the end of the file.

Reply:
:+1: